trio/_version.py

```diff
@@ -1,3 +1,3 @@
-# This file is imported from __init__.py and exec'd from setup.py
+# This file is imported from __init__.py and ast parsed by setuptools
```

Comment:
Typo?
```suggestion
# This file is imported from __init__.py and parsed by setuptools
```

Reply:
Not a typo I meant ast parsed

Reply:
I wanted to say something instead of eval/exec describing what setuptools does here